snuba/web/db_query.py

```diff
@@ -154,8 +152,8 @@ def execute_query(
     formatted_query: FormattedQuery,
     reader: Reader,
     timer: Timer,
-    stats: MutableMapping[str, Any],
-    clickhouse_query_settings: MutableMapping[str, Any],
+    stats: Dict[str, Any],
```

Comment:
why the change from MutableMapping to Dict?

Reply:
Typing general principle is to be as relaxed as possible for inputs and strict as possible for outputs.

And if you are going to use `Dict`, please use `dict` instead.

Reply:
I undid these changes because they're just adding noise